shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/base/BaseMySQLITCase.java

```diff
@@ -15,54 +15,61 @@
 
 import javax.xml.bind.JAXB;
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
-import java.util.List;
-import java.util.Objects;
 import java.util.Properties;
 
 public abstract class BaseMySQLITCase extends BaseITCase {
     
-    protected static final DatabaseType DATABASE = new MySQLDatabaseType();
+    protected static final DatabaseType DATABASE_TYPE = new MySQLDatabaseType();
     
     private final ExtraSQLCommand extraSQLCommand;
     
+    @Getter
+    private final SQLHelper sqlHelper;
+    
     public BaseMySQLITCase(final ScalingParameterized parameterized) {
         super(parameterized);
-        extraSQLCommand = JAXB.unmarshal(Objects.requireNonNull(BasePostgreSQLITCase.class.getClassLoader().getResource(parameterized.getParentPath() + "/sql.xml")), ExtraSQLCommand.class);
-        initTableAndData();
+        extraSQLCommand = JAXB.unmarshal(BaseMySQLITCase.class.getClassLoader().getResource(parameterized.getScenario()), ExtraSQLCommand.class);
+        sqlHelper = new SQLHelper(DATABASE_TYPE, extraSQLCommand, getJdbcTemplate());
     }
     
-    @SneakyThrows({SQLException.class, InterruptedException.class})
-    protected void initTableAndData() {
+    @SneakyThrows(SQLException.class)
+    protected void addResource() {
         Properties queryProps = createQueryProperties();
         // TODO if use jdbcurl like "jdbc:mysql:localhost:3307/sharding_db", will throw exception show "Datasource or ShardingSphere rule does not exist"
         try (Connection connection = DriverManager.getConnection(JDBC_URL_APPENDER.appendQueryProperties(getComposedContainer().getProxyJdbcUrl(""), queryProps), "root", "root")) {
             connection.createStatement().execute("USE sharding_db");
             addResource(connection);
         }
-        initShardingRule();
-        setIncreaseTaskThread(new Thread(new MySQLIncrementTaskRunnable(getJdbcTemplate(), extraSQLCommand)));
-        getJdbcTemplate().execute(extraSQLCommand.getCreateTableOrder());
-        getJdbcTemplate().execute(extraSQLCommand.getCreateTableOrderItem());
+    }
+    
+    protected void startIncrementTask(final KeyGenerateAlgorithm keyGenerateAlgorithm) {
+        setIncreaseTaskThread(new Thread(new MySQLIncrementTaskRunnable(getJdbcTemplate(), extraSQLCommand, keyGenerateAlgorithm)));
         getIncreaseTaskThread().start();
-        Pair<List<Object[]>, List<Object[]>> dataPair = TableCrudUtil.generateMySQLInsertDataList(3000);
-        getJdbcTemplate().batchUpdate(extraSQLCommand.getFullInsertOrder(), dataPair.getLeft());
-        getJdbcTemplate().batchUpdate(extraSQLCommand.getInsertOrderItem(), dataPair.getRight());
+    }
+    
+    /**
+     * Add no use table, to test part of the table.
+     */
+    protected void addNoUseTable() {
+        getJdbcTemplate().execute("CREATE TABLE no_use(id int(11) NOT NULL)");
+        getJdbcTemplate().execute("INSERT INTO no_use(id) values (1)");
     }
```

Comment:
Seems there's no `no_use` table sharding rule, it won't be handled as partial tables scaling.